content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Manual rotation of CA certificates
+content_template: templates/task
+reviewers:
+- 
+---
+
+{{% capture overview %}}
+
+This page shows how to manually rotate certificate authority (CA) certificates.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
```

Comment:
Do these steps assume that you have opted for a single root CA deployment as per https://kubernetes.io/docs/setup/best-practices/certificates/#single-root-ca? If so, I would mention that.

Reply:
@sftim no, it doesn't assume that.

Reply:
:ok_hand: